modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderIT.java

```diff
@@ -72,7 +74,8 @@
     protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
         Settings.Builder settings = Settings.builder().put(super.nodeSettings(nodeOrdinal, otherSettings));
         if (ENDPOINT != null) {
-            settings.put(GeoIpDownloader.ENDPOINT_SETTING.getKey(), ENDPOINT);
+            String endpoint = getTestName().endsWith("Cli") ? ENDPOINT + "cli/overview.json" : ENDPOINT;
```

Comment:
maybe set/overwrite the custom endpoint for cli files in the `testGeoIpDatabasesDownloadCli()` test instead of here with a conditional? I think it is clearer to have this logic where it is used.

Reply:
I refactored CLI case to separate class so it's cleaner now